### PR TITLE
Fix Footer Issue On Medium Viewports

### LINKED ISSUE #1634: Footer margin adjustment needed?
Raised by a RepoSense dev [here](https://github.com/reposense/RepoSense/pull/1559#discussion_r667010276), regarding some lines in the main css file generated by MarkBind `init`.

image

### PR DESCRIPTION
**Pull request by kaixin-hc**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [X] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

Fixes #1634 

**Overview of changes:**
Remove the margin-left of the footer in the default CSS template.

**Anything you'd like to highlight / discuss:**

**Testing instructions:**
Run `markdown init` and resize the viewport; there should be no left margin for the footer at any point.

**Proposed commit message: (wrap lines at 72 characters)**

```
Fix footer margin on medium viewports
```

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

File: packages/cli/test/functional/test_site_templates/test_default/expected/stylesheets/main.css [hunk 1]
```diff
@@ -110,10 +110,6 @@ mark {
     #site-nav {
         display: none;
     }
-
-    footer {
-        margin-left: 60px;
-    }
 }
```
Comment (jonahtanjz) [inline packages/cli/test/functional/test_site_templates/test_default/expected/stylesheets/main.css:113]: Only the changes in this file need to be committed as modifying the css file, `packages/core/template/default/stylesheets/main.css`, will only affect this test file.

**Review by jonahtanjz — COMMENTED** [review]
Nice work @kaixin-hc 👍 

Generally looks good. I've tried it out and it seems to have fixed the issue.

I believe only one of the expected test files needs to be updated. The rest of the changes in the other test files can be stashed as they are not relevant to the PR.

File: packages/cli/test/functional/test_site_templates/test_default/expected/stylesheets/main.css (hunk 1 above)
Comment (kaixin-hc) [reply packages/cli/test/functional/test_site_templates/test_default/expected/stylesheets/main.css:113]: Thanks for checking this for me! I've reverted and pushed changes to just that file.

**Comment by jonahtanjz** [comment]
Thank you for updating :) It looks good!

Sorry missed this out earlier, seems like the css for the documentation site has the same issue. 

image

While you are at it, able to also remove the `margin-left`s from the documentation css `docs/css/main.css`, line 114 and 136, as well?

**Comment by kaixin-hc** [comment]
Sorry, for the wait, I've added it!

**Review by jonahtanjz — APPROVED** [review]
Thank you 👍 

LGTM!

**Pull request merged**